### ISSUE TITLE
Add more information about database properties

### ISSUE THREAD

**Issue by rafalkasa** [schemaspy/schemaspy#9]

**Comment by npetzall** [comment]
There's some documentation for dbTypes in PR #183 
I could probably split the documentation to a separate PR if that helps.

**Comment by npetzall** [comment]
@rafalkasa Good to close or do you know if any additional documentation that is needed?

**Comment by rafalkasa** [comment]
@npetzall that issue is about extend the SchemaSpy to retrieve more database properties like:

- Size
- Owner
- Creation date
- Last database backup date
- Number of open connection
- Collation / Encoding
- Database Files size
- File groups

In current application we only retrieved:
![image](https://user-images.githubusercontent.com/19885116/33771152-4511de7a-dc30-11e7-9223-f7da9b698cdd.png)

**Comment by npetzall** [comment]
Ahh now it clear.

**Comment by rafalkasa** [comment]
yes open connection is maybe the not good one, but there are only examples :smile:
